package.json

```diff
@@ -24,7 +24,7 @@
     "compile:grav": "webpack --display-entrypoints --config webpack.grav.config.js",
     "lint:js": "eslint ./",
     "lint:scss": "stylelint '{source, apps/drupal, apps/pl}/**/*.scss'",
-    "lint": "npm run lint:js & npm run lint:scss",
+    "lint": "npm run lint:js && npm run lint:scss",
```

Comment:
Is it on purpose that if JS linting fails it don't bother with SCSS linting? Alternatively could use a semicolon to run it in sequence, failing first only if the script it executed in a context where an error causes a script bail.

Reply:
A quick note on this: Travis was exploding due to the simultaneous nature of the `&`. Moving to consecutive commands makes Travis happy, for some reason.

Regarding semicolon syntax, like this?

```sh
npm run lint:js;npm run lint:scss
```

Reply:
Si. Though conventionally there's a space after the semicolon.

Reply:
Moved to `;`